src/app/voting/voting-element/voting-card/voting-card.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="card-holder">
+  <mat-card class="card">
+    <p style="margin: unset">{{ name }}</p>
+    <img src="favicon.ico" height="80px" class="cover" />
+    <p>{{ text }}</p>
+    <div class="flex alignment">
+      <button mat-raised-button class="button" (click)="closeCard()">
+        Close
+      </button>
```

Comment:
```suggestion
      <button mat-raised-button class="btn btn-primary" (click)="closeCard()">
        Close
      </button>
```

Apply global styles.